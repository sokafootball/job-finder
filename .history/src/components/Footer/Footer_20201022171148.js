import React from 'react';
import './Footer.css';

function Footer(){
  return (
    <div>
      <p>A page by <span id="signature">Laurent Capello</span></p>
    </div>
  )
}

export default Footer
