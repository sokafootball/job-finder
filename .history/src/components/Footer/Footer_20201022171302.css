#footer{
  margin: 20px 0px 30px 0px;
}
