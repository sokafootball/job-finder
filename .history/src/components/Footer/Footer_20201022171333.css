#footer{
  font-size: 22px;
  margin: 20px 0px 30px 0px;
}
