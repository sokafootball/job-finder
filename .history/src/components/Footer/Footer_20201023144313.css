#footer{
  font-size: 22px;
  margin: 20px 0px 30px 0px;
}

#signature{
  text-shadow: var(--border-color) 2px 2px;
  font-size: 25px;
  font-weight: 800;
}
