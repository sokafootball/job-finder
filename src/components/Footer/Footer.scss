@import "../../shared/variables.scss";

.footer{
  font-size: 22px;
  margin: auto 0px 30px 0px;
  &__signature{
    color:$job-title-color;
    text-shadow: $border-color 2px 2px;
    font-size: 25px;
    font-weight: 800;
  }
}

