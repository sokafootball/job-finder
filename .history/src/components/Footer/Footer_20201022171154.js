import React from 'react';
import './Footer.css';

function Footer(){
  return (
    <p>A page by <span id="signature">Laurent Capello</span></p>
  )
}

export default Footer
