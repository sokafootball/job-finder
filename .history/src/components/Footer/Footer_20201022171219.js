import React from 'react';
import './Footer.css';

function Footer(){
  return (
    <p id="footer">A page by <span id="signature">Laurent Capello</span></p>
  )
}

export default Footer
