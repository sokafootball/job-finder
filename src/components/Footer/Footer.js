import './Footer.scss'
import React from 'react'

function Footer() {
  return (
    <p className="footer">
      A page by <span className="footer__signature">Laurent Capello</span>
    </p>
  )
}

export default Footer
