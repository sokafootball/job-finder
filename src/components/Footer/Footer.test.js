import { shallow } from 'enzyme'
import Footer from './Footer'
import React from 'react'

describe('Footer', () => {
  let wrapped = shallow(<Footer />)
  it('renders the Footer component correctly', () => {
    expect(wrapped).toMatchSnapshot()
  })
})
