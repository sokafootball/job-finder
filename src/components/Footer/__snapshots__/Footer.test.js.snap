// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the Footer component correctly 1`] = `
<p
  className="footer"
>
  A page by 
  <span
    className="footer__signature"
  >
    Laurent Capello
  </span>
</p>
`;
